### ISSUE TITLE
Stacking Regressor

### ISSUE THREAD

**Issue by KuriaMaingi** [Ibotta/pure-predict#13]
Hi, could you kindly add support for converting Stacking Regressors?

**Comment by denver1117** [comment]
Yes unfortunately this meta-estimator isn't supported. By design, pure predict only supports a subset of sklearn estimators and meta-estimators. Please feel free to contribute, I'd happily comment on and approve a PR that implements this solution.
